src/styles/List.css

```diff
@@ -11,6 +11,8 @@
 .ul-items {
   padding-left: 0rem;
   list-style: none;
+  height: 26rem;
```

Comment:
I wonder if `vh` (i.e. something like `50vh` which would mean 50% of the viewport height) might be a better unit to use here. `26rem` might mean your nav is off the screen on smaller screen sizes.

Reply:
I actually tried doing that, but then `overflow: scroll` wasn't working.